src/ripple/core/ThreadEntry.h

```diff
@@ -30,6 +30,16 @@
 
 namespace ripple
 {
+
+#ifndef NO_LOG_UNHANDLED_EXCEPTIONS
+namespace detail
+{
+void setThreadName(std::string name);
+}
+
+void terminateHandler();
+#endif
+
 /**
 Report uncaught exceptions to DebugLog and cerr
 
```

Comment:
Note that this comment block needs tending to.  In particular the paragraph starting on line 59 (regarding the lambda) should be removed.